### PR TITLE
docs: add rbac docs

### PR DESCRIPTION
**Pull request by langchain-infra**

### CODE REVIEW

File: versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Setup RBAC
```
Comment (agola11) [inline versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx:1]:
```suggestion
# Set up RBAC
```

File: versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+:::note
+RBAC (Role-Based Access Control) is a feature that is only available to Enterprise customers. If you are interested in this feature, please contact our sales team at sales@langchain.dev
+Other plans default to using the Admin role for all users. Read more about roles under [admin concepts](../../concepts/admin/admin.mdx)
+:::
+
+LangSmith relies on RBAC to manage user permissions. This allows you to control who can access your LangSmith workspace and what they can do within the account.
+Only users with the `workspace:manage` permission can can manage access control.
+
+## Creating a Role
```
Comment (agola11) [inline versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx:11]:
```suggestion
## Create a role
```

**Review by agola11 — COMMENTED** [review]
would leave second word lower cased in titles to stick with conventions

File: versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+While there are several default roles that come with LangSmith, you can create custom roles to suit your needs.
```
Comment (agola11) [inline versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx:13]: list the roles

File: versioned_docs/version-2.0/how_to_guides/index.md [hunk 4]
```diff
@@ -11,9 +11,9 @@ Step-by-step guides that cover key tasks and operations in LangSmith.
 
 - [Create an account and API key](./how_to_guides/setup/create_account_api_key)
 - [Create an organization](./how_to_guides/setup/create_organization)
-- [Setup billing](./how_to_guides/setup/setup_billing)
+- [Set up billing](./how_to_guides/setup/set_up_billing)
 - [Set up a workspace](./how_to_guides/setup/set_up_workspace)
-- [ ] Assign roles (enterprise only)
+- [Set up access control (enterprise only)](./how_to_guides/setup/set_up_access_control)
```
Comment (agola11) [inline versioned_docs/version-2.0/how_to_guides/index.md:16]: inline H2 fields in the page as tabbed bullets under here (see other sections for examples)

**Review by agola11 — APPROVED** [review]

File: versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx (hunk 3 above)
Comment (langchain-infra) [reply versioned_docs/version-2.0/how_to_guides/setup/set_up_access_control.mdx:13]: would that be redundant given that we already have these listed under concepts?

**Pull request merged**